source/core/src/main/com/csse3200/game/areas/ForestGameArea.java

```diff
@@ -122,10 +122,12 @@ public void create() {
       player.getEvents().addListener("defeatLandBoss", this::spawnKangarooBoss);
```

Comment:
these events are reserved for quest completion upon boss death, please use a new event name